code/modules/cards/deck/deck.dm

```diff
@@ -0,0 +1,170 @@
+#define DECK_SHUFFLE_TIME 5 SECONDS
+#define DECK_SYNDIE_SHUFFLE_TIME 3 SECONDS
+
+/obj/item/toy/cards/deck
+	name = "deck of cards"
+	desc = "A deck of space-grade playing cards."
+	icon = 'icons/obj/playing_cards.dmi'
+	icon_state = "deck_nanotrasen_full"
+	w_class = WEIGHT_CLASS_SMALL
+	worn_icon_state = "card"
+	/// The amount of time it takes to shuffle
+	var/shuffle_time = DECK_SHUFFLE_TIME
+	/// Deck shuffling cooldown.
+	COOLDOWN_DECLARE(shuffle_cooldown)
+	/// Tracks holodeck cards, since they shouldn't be infinite
+	var/obj/machinery/computer/holodeck/holo = null
+	/// If the deck is the standard 52 playing card deck (used for poker and blackjack)
+	var/is_standard_deck = TRUE
+	/// The amount of cards to spawn in the deck (optional)
+	var/decksize = INFINITY
+	///Wielding status for holding with two hands
+	var/wielded = FALSE
+
+	// the below vars will be inherited by the singlecards spawned in the deck
+	hitsound = null
```

Comment:
I'm aware of this however this is a unique exception since all new singlecards will inherit the vars under that line. Which is important for others to know if they decide to make a new deck in the future.